internal/applicationsnapshot/templates/_results.tmpl

```diff
@@ -16,7 +16,8 @@
     {{- colorIndicator $type }} {{ colorText $type (printf "[%s] %s" $type .Metadata.code) }}{{ nl -}}
 
     {{- if $imageRef -}}
-      {{- indentWrap $indent $wrap (printf "ImageRef: %s" $imageRef ) }}{{ nl -}}
+      {{/* Keep the indent but don't actually wrap the text for this one */}}
+      {{- indentWrap $indent 1000 (printf "ImageRef: %s" $imageRef ) }}{{ nl -}}
```

Comment:
Would this work?
```suggestion
      ImageRef: {{  $imageRef }}{{ nl -}}
```

Reply:
Actually it's not quite so simple since the space before gets eaten up. I tried a few variations, decide to add an indent helper that just indents. :shrug:

Reply:
Doesn't matter much, but I was thinking about the $indent var and avoiding hard coded indenting...